### ISSUE TITLE
[Feature Request] Add the option for replacing thumbnails with actual video frames

### ISSUE THREAD

**Issue by TotalCaesar659** [TeamNewPipe/NewPipe#2505]
Many video uploaders make very awful thumbnails (they're bright, have BIG TEXT, don't reflect actual content of the video etc.)
[Clickbait Remover for Youtube](https://github.com/pietervanheijningen/clickbait-remover-for-youtube) lets you replace these thumbnails with actual frames from beginning, middle or the end of the video.
Could you add same option to NewPipe?

**Comment by Stypox** [comment]
@nv95 this could be considered when you are going to implement frames into the NewPipe app.  What do you think?

**Comment by opusforlife2** [comment]
Wouldn't this require automatically playing the video first and then capturing a frame from it? If so, is it worth it?

**Comment by Stypox** [comment]
@opusforlife2 this has already been implemented in the extractor (search for "Frames"). Youtube provides lightweight frames separately from video

**Comment by opusforlife2** [comment]
Ah. So, feasible now?

**Comment by designstorming** [comment]
Bumping this issue up, again.
Are there plans to implement the feature described in the title?

Today's creators' upload thumbnails that are mostly attention diverting and false. By opening the feed (or search results or any other page with video list), (NewPipe, Youtube...) users are presented with attention grabbing (content noise) images that advertise either "urgency", or are clickbait.

There is already a chrome extension called [Clickbait Remover for Youtube](https://chrome.google.com/webstore/detail/clickbait-remover-for-you/omoinegiohhgbikclijaniebjpkeopip) that eases user experience on this platform. I think it would help users to filter-out "trash" content.

I think this has no influence on advertising and NewPipe's [position on it](https://newpipe.net/blog/pinned/newpipe-and-online-advertising/).

Thanks for taking the time to read this feedback.

**Comment by Stypox** [comment]
Getting back with more information: this is doable even without frames. We would just need to choose a different thumbnail. AudricV's images PR would come in handy.

**Comment by GameOverFlowChart** [comment]
Yes please, I used YouTube before it was bought by Google. Before it was flooded with videos where YouTubers make childish facial expressions for the thumbnails. The best solution would be a random frame somewhere from the middle. These facial expressions are nightmare fuel.
